starter-code/css/main.css

```diff
@@ -0,0 +1,98 @@
+* {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+}
+
+body {
+    width: 900px;
+    margin: 0 auto;
+}
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 65px;
+    padding: 0px;
+    margin: 10px 0px;
+}
+
+    h1 a {
+        color: #f8ce78;
+        letter-spacing: -2px;
+    }
+     
+h2 {
+    padding: 0px;
+    margin: 10px 0px;
+    font-size: 25px;
+}
+
+img { width: 100%;}
+article {
+    width: 850px;

```

Comment:
Again, set the width with a % so that you are not limiting your site to break on a mobile device.